WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostCardActionsHandler.kt

```diff
@@ -141,8 +141,44 @@ class ReaderPostCardActionsHandler @Inject constructor(
         }
     }
 
+    suspend fun handleFollowRecommendedSiteClicked(blogId: Long, feedId: Long?, isAskingToFollow: Boolean) {
```

Comment:
For now, I did copy-paste the code from another function but I think this could be unified

Reply:
Yep, I think they can be unified.